### PR TITLE
Minor fixes (task #1472)

### PR DESCRIPTION
**Pull request by flangofas**

### CODE REVIEW

File: config/file_storage.php [hunk 1]
```diff
@@ -49,8 +49,8 @@
                 'mode' => 'inbound',
                 //Ratio 16:9
                 //4 Columns based on Bootstrap 3
-                'width' => 230,
-                'height' => 129
+                'width' => 262.5,
```
Comment (mamchenkov) [inline config/file_storage.php]: @flangofas aren't these in pixels?

Comment (flangofas) [inline config/file_storage.php]: @mamchenkov yes :+1:

Comment (mamchenkov) [inline config/file_storage.php]: @flangofas how's the image resized to 262.5 pixels? :)

Comment (flangofas) [inline config/file_storage.php]: @mamchenkov I use Bootstrap's columns to set the dimensions. It's done by the [Imagine](https://github.com/burzum/cakephp-imagine-plugin) plugin

Comment (mamchenkov) [inline config/file_storage.php]: @flangofas maybe I'm not understanding something here ... if Imagine plugin creates the thumbnails, then it's probably passing the dimension to the image manipulation library like GD or ImageMagick.  Those probably don't support fractional pixels. Bootstrap is CSS, which is different - the browser takes care of rounding and rendering those.

Comment (flangofas) [inline config/file_storage.php]: @mamchenkov sure. I'll round it up. :)

Comment (flangofas) [inline config/file_storage.php]: @mamchenkov Done. Please check again.

**Comment by mamchenkov** [comment]
:+1:

**Pull request merged**